apps/web/src/forms/SymptomChecker.tsx

```diff
@@ -63,65 +58,108 @@ export const SymptomChecker = () => {
     }
   };
 
-  const pages: ReactElement[] = [
-    <Q1SevereSymptom key={0} onChange={onChange} />,
-    <Q2DifficultBreathing key={1} onChange={onChange} />,
-    <Q3Symptoms key={2} values={values} onChange={onChange} />,
-    <Q3SymptomBreathingSeverity key={3} values={values} onChange={onChange} />,
-    <Q3SymptomCoughSeverity key={4} values={values} onChange={onChange} />,
-    <Q3SymptomBodyAchesSeverity key={5} values={values} onChange={onChange} />,
-    <Q3SymptomSoreThroatSeverity key={6} values={values} onChange={onChange} />,
-    <Q4TestResult key={100} values={values} onChange={onChange} />,
-  ];
+  const decideNextPage = (values: Partial<SymptomCheckerForm>) => {
```

Comment:
do we really want routes for each outcome? Not sure how it will be useful

Reply:
I have updated based on existing things, we can change this conditional renders  once the happy path is done.